core/services/fluxmonitor/fetchers.go

```diff
@@ -185,9 +185,12 @@ func (m *medianFetcher) Fetch() (decimal.Decimal, error) {
 		}
 	}
 
-	errorRate := float64(len(fetchErrors)) / float64(len(m.fetchers))
+	fetchersCount := len(m.fetchers)
+	fetchErrorsCount := len(fetchErrors)
+	errorRate := float64(fetchErrorsCount) / float64(fetchersCount)
 	if errorRate >= 0.5 {
-		return decimal.Decimal{}, errors.Wrap(multierr.Combine(fetchErrors...), "majority of fetchers in median failed")
+		err := errors.Wrap(multierr.Combine(fetchErrors...), fmt.Sprintf("at least 50%% of the fetchers in median failed (%d/%d)", fetchErrorsCount, fetchersCount))
```

Comment:
Why not put the rate in the error message? Instead of saying `at least 50% of..` just say `67% of fetchers failed`

Reply:
The rate is available as a fraction `(5/10)` in the new message. It also has the added benefit that it displays the count of fetchers and the count of failures all in a single form. Do you think it's also useful to calculate and display the percentage?

The major intent of this PR change is to update the condition message. The old message `majority of fetchers in median failed` is not quite correct. If 1/2 fetchers fail it will display this message. But 1/2 is not the `majority`. The new condition message should capture this `at least 50% of the fetchers in median failed` but lmk if you think it's still not clear enough.